package-lock.json

```diff
@@ -0,0 +1,327 @@
+{
```

Comment:
Our project uses yarn to manage dependencies. Please don't commit a new package-lock file.